Expecto/Expecto.fs

```diff
@@ -713,32 +751,68 @@ module Impl =
       filter   : Test -> Test
       /// Allows the test printer to be parametised to your liking.
       printer : TestPrinters
-      /// Verbosity level (default: Info)
+      /// Verbosity level (default: Info).
       verbosity : LogLevel
       /// Optional function used for finding source code location of test
-      /// Defaults to empty source code
+      /// Defaults to empty source code.
       locate : TestCode -> SourceLocation
+      /// FsCheck maximum number of tests (default: 100).
+      maxTest: int
+      /// FsCheck start size (default: 1).
+      startSize: int
+      /// FsCheck end size (default: 100).
+      endSize: int
     }
+    static member Default =
```

Comment:
Yes it was in the Tests module. I need it in further up so I put it on the type. There is still a property on Tests that links to this. I've renamed to defaultConfig.